packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextLayoutManager.java

```diff
@@ -146,6 +146,29 @@ public static boolean isRTL(MapBuffer attributedString) {
         == LayoutDirection.RTL;
   }
 
+  private static int getTextJustificationMode(MapBuffer attributedString) {
+    int justificationMode = (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) ? 0 : Layout.JUSTIFICATION_MODE_NONE;
+    if (!attributedString.contains(AS_KEY_FRAGMENTS)) {
+      return justificationMode;
+    }
+
+    MapBuffer fragments = attributedString.getMapBuffer(AS_KEY_FRAGMENTS);
+    if (fragments.getCount() != 0) {
+      MapBuffer fragment = fragments.getMapBuffer(0);
+      MapBuffer textAttributes = fragment.getMapBuffer(FR_KEY_TEXT_ATTRIBUTES);
+
+      if (textAttributes.contains(TextAttributeProps.TA_KEY_ALIGNMENT)) {
+        String alignmentAttr = textAttributes.getString(TextAttributeProps.TA_KEY_ALIGNMENT);
+
+        if (alignmentAttr.equals("justified")) {
+          justificationMode = (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) ? 1 : Layout.JUSTIFICATION_MODE_INTER_WORD;
+        }
+      }
+    }
+
+    return justificationMode;
+  }
+
```

Comment:
I think we could merge retrieving `alignmentAttr` into one function as similar code is used in `getTextAlignment`